sktime/datasets/_single_problem_loaders.py

```diff
@@ -70,35 +73,41 @@ def load_UCR_UEA_dataset(name, split=None, return_X_y=True, extract_path=None):
         this function will look in the extract_path first, and if it is not present,
         attempt to download the data from www.timeseriesclassification.com, saving it to
         the extract_path.
-    split : None or str{"train", "test"}, optional (default=None)
+    split: None or str{"train", "test"}, optional (default=None)
         Whether to load the train or test partition of the problem. By default it
-        loads both into a single dataset, otherwise it looks only for files of the
-        format <name>_TRAIN.ts or <name>_TEST.ts.
-    return_X_y : bool, optional (default=False)
-        it returns two objects, if False, it appends the class labels to the dataframe.
-    extract_path : str, optional (default=None)
-        the path to look for the data. If no path is provided, the function
-        looks in `sktime/datasets/data/`.
+        loads both. into a single data structure, otherwise it looks only for files
+        of the format <name>_TRAIN.ts or <name>_TEST.ts.
+    return_X_y: bool, optional (default=True)
+        If True, returns (features, target) separately instead of a single
+        dataframe with columns for features and the target.
+    return_type: None or str{"numpy2d", "numpyflat", "numpy3d", "nested_univ"},
+    optional (default=None). Controls the returned data structure.
 
     Returns
     -------
-    X: pandas DataFrame
-        The time series data for the problem with n_cases rows and either
-        n_dimensions or n_dimensions+1 columns. Columns 1 to n_dimensions are the
-        series associated with each case. If return_X_y is False, column
-        n_dimensions+1 contains the class labels/target variable.
-    y: numpy array, optional
-        The class labels for each case in X, returned separately if return_X_y is
-        True, or appended to X if False
+    X:  The time series data for the problem. If return_type is either
```

Comment:
incomplete/incorrect docstring
behaviour does not seem to be accurate? what happens if "split"?
extract_path missing

Reply:
added extract path, not sure what "what happens if "split"?" means